src/pudl/package_data/notebooks/how_to_access_pudl.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>It might be helpful to write a blurb about why one might access the data this way for each of the different methods. What are the benefits of each?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>